apkid/rules/dex/packers.yara

```diff
@@ -243,3 +243,79 @@ rule jar_pack01 : packer
     is_dex and
     ($pre_jar and $jar_data and $post_jar)
 }
+
+rule unk_packer_a : packer
+{
+
+  meta:
+    description = "Unknown packer"
```

Comment:
The rule is excellent; it only needs a good name. It doesn't really matter, as long as it's unique. Something like:`GaoXor` (part of the obfuscated method name and Xor)

Reply:
As you wish man

Reply:
If we don't name them, we'll have Unknown packer 1, Unknown packer 2, etc. for days